rest-server/src/main/java/io/github/nubesgen/configuration/NubesgenConfiguration.java

```diff
@@ -203,11 +205,23 @@ public boolean isRuntimeJava() {
             RuntimeType.SPRING.equals(this.getRuntimeType()) ||
             RuntimeType.SPRING_GRADLE.equals(this.getRuntimeType()) ||
             RuntimeType.QUARKUS.equals(this.getRuntimeType()) ||
+            RuntimeType.MICRONAUT.equals(this.getRuntimeType()) ||
```

Comment:
should not this be better as a method of `RuntimeType`. e.g. `RuntimeType::isJava?`

Reply:
this would mean refactoring all the other types, so it would go beyond that PR, let's not do this yet